tracker/tracker_object.rst

```diff
@@ -206,24 +206,51 @@ Custom variables
 Custom dimensions
 -----------------
 
-.. function:: setCustomDimension(customDimensionId, customDimensionValue)
+.. function:: setCustomDimensionValue(customDimensionId, customDimensionValue)
+
+    .. versionadded:: 15.3
 
     The function that sets a custom dimension to be used later.
 
     :param string customDimensionId: **Required** Id of custom dimension.
     :param string customDimensionValue: **Required** Value of custom dimension.
 
+.. function:: setCustomDimension(customDimensionId, customDimensionValue)
+
+    .. deprecated:: 15.3
+        Function :func:`setCustomDimension` is deprecated due to difficulty of use (passed values should be URL encoded). Please use :func:`setCustomDimensionValue` instead.
+
+    The function that sets a custom dimension to be used later.
```

Comment:
```suggestion
    Function that sets a custom dimension to be used later.
```
Somehow I have a feeling that there should be no article at the beginning of this sentence (or maybe rather `a` instead of `the`), but I'm not sure. Consult someone if you feel like it.